### PR TITLE
N29: Add Rusty Engine 3.0 section

### PR DESCRIPTION
**Pull request by CleanCut**
☝🏻 

_Part of #888_

### CODE REVIEW

File: content/news/029/index.md [hunk 1]
```diff
@@ -69,6 +69,26 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Engine Updates
 
+### [Rusty Engine 3.0]
+
+![image/GIF description](rusty_engine3.png)
```
Comment (17cupsofcoffee) [inline content/news/029/index.md:74]:
```suggestion
![An example Rusty Engine game](rusty_engine3.png)
```

**Review by 17cupsofcoffee — APPROVED** [review]

File: content/news/029/index.md [hunk 2]
```diff
@@ -69,6 +69,26 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
 ## Engine Updates
 
+### [Rusty Engine 3.0]
+
+![An example Rusty Engine game](rusty_engine3.png)
+_The "Road Race" game prototype running under Rusty Engine 3.0_
+
+[Rusty Engine] by [Nathan Stocks] is a game engine for people who are learning Rust.
+
+Version 3.0 is a large release with many breaking changes. Notable new features include: [a full tutorial], custom asset loading (sprites, sounds, fonts), customizable game state, an interactive collider creator, and much more. See [the changelog for 3.0] for the full details.
```
Comment (17cupsofcoffee) [inline content/news/029/index.md:79]:
```suggestion
Version 3.0 is a large release with many breaking changes. Notable new features
include: [a full tutorial], custom asset loading (sprites, sounds, fonts), 
customizable game state, an interactive collider creator, and much more.
See [the changelog for 3.0] for the full details.
```

File: content/news/029/index.md [hunk 3]
```diff
@@ -69,6 +69,26 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+[twitter](https://twitter.com/nathanstocks/status/1476724623903367168)_
+
+[Rusty Engine 3.0]: https://github.com/CleanCut/rusty_engine/blob/main/CHANGELOG.md#300---2021-12-30
+[Rusty Engine]: https://github.com/CleanCut/rusty_engine
+[a full tutorial]: https://cleancut.github.io/rusty_engine/
+[the changelog for 3.0]: https://github.com/CleanCut/rusty_engine/blob/main/CHANGELOG.md#300---2021-12-30
+[Nathan Stocks]: https://github.com/CleanCut
+
+
 ## Learning Material Updates
```
Comment (17cupsofcoffee) [inline content/news/029/index.md:92]:
```suggestion
[Nathan Stocks]: https://github.com/CleanCut

## Learning Material Updates
```

File: content/news/029/index.md (hunk 2 above)
Comment (CleanCut) [reply content/news/029/index.md]: Ah, I have my editor set to soft-wrap markdown. Forgot about explicitly adding the breaks. 🙈

**Pull request merged**